### ISSUE TITLE
Missing handler to convert remote paths to local paths on Goto command

### ISSUE THREAD

**Issue by DamnWidget** [DamnWidget/anaconda#591]
## Expected Behaviour
When using a remote interpreter, and a valid remote path mapping configuration is in use, Goto **must** Work 

## Actual Behaviour
It fails miserably complaining about path not existing (uses remote path so it does not really exists)

## Steps to Reproduce
Just start a remote worker (vagrant, docker or just remote), set a paths map and use Goto

## ST3, Anaconda and OS versions
Any

## ST3 Console Logs
```
FileNotFoundError: [Errno 2] No such file or directory: '/path/on/vagrant/machine/to/file.py'
```

## Related with
This issue is related with https://github.com/DamnWidget/anaconda/issues/588#issuecomment-262251209

**Comment by DamnWidget** [comment]
@kiddick could you confirm that this is actually fixed please?

Thanks.

**Comment by kiddick** [comment]
Error while loading anaconda:

```
reloading plugin Anaconda.anaconda
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 109, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.3/importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 584, in _check_name_wrapper
  File "<frozen importlib._bootstrap>", line 1022, in load_module
  File "<frozen importlib._bootstrap>", line 1003, in load_module
  File "<frozen importlib._bootstrap>", line 560, in module_for_loader_wrapper
  File "<frozen importlib._bootstrap>", line 868, in _load_module
  File "<frozen importlib._bootstrap>", line 313, in _call_with_frames_removed
  File "/home/me/.config/sublime-text-3/Packages/Anaconda/anaconda.py", line 21, in <module>
    from .commands import *
  File "/home/me/.config/sublime-text-3/Packages/Anaconda/commands/__init__.py", line 17, in <module>
    from .goto import (
  File "/home/me/.config/sublime-text-3/Packages/Anaconda/commands/goto.py", line 11, in <module>
    from ..anaconda_lib.helper import is_remote_session, get_interpreter
ImportError: No module named 'Anaconda.anaconda_lib.helper'
```

**Comment by DamnWidget** [comment]
Has been fixed in a PR minutes ago https://github.com/DamnWidget/anaconda/pull/593

Try to reinstall anaconda

**Comment by kiddick** [comment]
I am getting this:
```
ERROR:root:'str' object has no attribute 'pathmap'
ERROR:root:Traceback (most recent call last):
ERROR:root:  File "/home/me/.config/sublime-text-3/Packages/Anaconda/anaconda_lib/jsonclient.py", line 104, in process_message
ERROR:root:    callback(data)
ERROR:root:  File "/home/me/.config/sublime-text-3/Packages/Anaconda/commands/goto.py", line 44, in on_success
ERROR:root:    path = self._infere_context_data(result[1])
ERROR:root:  File "/home/me/.config/sublime-text-3/Packages/Anaconda/commands/goto.py", line 61, in _infere_context_data
ERROR:root:    directory_map = interpreter.pathmap
```

**Comment by kiddick** [comment]
I am getting this:
```
ERROR:root:'str' object has no attribute 'pathmap'
ERROR:root:Traceback (most recent call last):
ERROR:root:  File "/home/me/.config/sublime-text-3/Packages/Anaconda/anaconda_lib/jsonclient.py", line 104, in process_message
ERROR:root:    callback(data)
ERROR:root:  File "/home/me/.config/sublime-text-3/Packages/Anaconda/commands/goto.py", line 44, in on_success
ERROR:root:    path = self._infere_context_data(result[1])
ERROR:root:  File "/home/me/.config/sublime-text-3/Packages/Anaconda/commands/goto.py", line 61, in _infere_context_data
ERROR:root:    directory_map = interpreter.pathmap
```

**Comment by DamnWidget** [comment]
Thanks, I will release the fix later today

**Comment by DamnWidget** [comment]
Fixed in v2.1.9
